mathics/builtin/arithmetic.py

```diff
@@ -1057,14 +1092,16 @@ class DirectedInfinity(SympyFunction):
     """
 
     rules = {
+        "DirectedInfinity[Indeterminate]":"Indeterminate",
         "DirectedInfinity[args___] ^ -1": "0",
         "0 * DirectedInfinity[args___]": "Message[Infinity::indet, Unevaluated[0 DirectedInfinity[args]]]; Indeterminate",
         "DirectedInfinity[a_?NumericQ] /; N[Abs[a]] != 1": "DirectedInfinity[a / Abs[a]]",
         "DirectedInfinity[a_] * DirectedInfinity[b_]": "DirectedInfinity[a*b]",
         "DirectedInfinity[] * DirectedInfinity[args___]": "DirectedInfinity[]",
         "DirectedInfinity[0]": "DirectedInfinity[]",
```

Comment:
flake8 correctly informs me that "DirectedInfinity[0]" appears twice. See below in line 1116. The second occurrence (that is, not this one), will be the one used. 

If that is what is _desired_ then it would be better to remove this line.